src/core_plugins/console/public/src/directives/sense_settings.js

```diff
@@ -16,6 +16,16 @@ require('ui/modules')
         $scope.kbnTopNav.close();
       };
 
+      const self = this;
+
+      function onEnter(event) {
+        if (event.which === 13) {
+          self.apply();
+        }
+      }
```

Comment:
interesting, thanks for the explanation.  I thought the main benefit of arrow functions was to avoid the binding or 'this' assignments, I didn't realize that people use it only for anonymous function situations.  I took a look at the airbnb style guide (in case we make a decision to use that) and it looks like it suggests arrow fn or .bind, but favors those both over saving a 'this' reference.  https://github.com/airbnb/javascript#naming--self-this

But since we haven't yet committed to using that style guide, so I'm good with it as is. :)